index.html

```diff
@@ -119,27 +112,46 @@ <h1 class="brand-heading">Breeze '16</h1>
     <section id="contact" class="container content-section text-center">
         <div class="row">
             <div class="col-lg-8 col-lg-offset-2">
-                <h2>Contact Breeze Team</h2>
-                <p>Feel free to email us!</p>
-                <p><a href="mailto:feedback@snu-breeze.com">feedback@snu-breeze.com</a>
-                </p>
+                <h1 class="section-name">Contact</h1>
                 <ul class="list-inline banner-social-buttons">
                     <li>
-                        <a href="https://twitter.com/rhnvrm" class="btn btn-default btn-lg"><i class="fa fa-twitter fa-fw"></i> <span class="network-name">Twitter</span></a>
+                        <a href="https://www.facebook.com/snu.breeze/" class="btn btn-default btn-lg" target="_blank"><i class="fa fa-facebook fa-fw"></i> <span class="network-name">Facebook</span></a>
                     </li>
                     <li>
-                        <a href="https://github.com/rhnvrm" class="btn btn-default btn-lg"><i class="fa fa-github fa-fw"></i> <span class="network-name">Github</span></a>
+                        <a href="https://www.instagram.com/breeze_snu/" class="btn btn-default btn-lg" target="_blank"><i class="fa fa-instagram fa-fw"></i> <span class="network-name">Instagram</span></a>
                     </li>
-                </ul>
-                <a href="#map" class="btn btn-circle page-scroll">
-                    <i class="fa fa-angle-double-down animated"></i>
-                </a>                
+                </ul>                              
             </div>
+        <div class="right-text col-xs-5">
```

Comment:
We don't need floating here. see this. http://v4-alpha.getbootstrap.com/layout/grid/ 

Let bootstrap handle the responsive part.